apps/evm/server/force_inclusion.go

```diff
@@ -0,0 +1,350 @@
+package server
+
+import (
+	"context"
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/evstack/ev-node/core/da"
+	"github.com/evstack/ev-node/pkg/config"
+	"github.com/evstack/ev-node/pkg/genesis"
+	"github.com/evstack/ev-node/types"
+)
+
+const (
+	defaultForceInclusionAddr = "127.0.0.1:8547"
+	defaultReadTimeout        = 30 * time.Second
+	defaultWriteTimeout       = 30 * time.Second
+)
+
+// ForceInclusionServer provides an Ethereum-compatible JSON-RPC endpoint
+// that accepts transactions and submits them directly to the DA layer for force inclusion
+type ForceInclusionServer struct {
+	server          *http.Server
+	daClient        da.DA
+	config          config.Config
+	genesis         genesis.Genesis
+	logger          zerolog.Logger
+	namespace       []byte
+	executionRPCURL string
+	httpClient      *http.Client
+}
+
+// NewForceInclusionServer creates a new force inclusion server
+func NewForceInclusionServer(
```

Comment:
i dont fully follow the need for the server, can we send it without the server? I could be misunderstanding the flow

Reply:
That server is the RPC endpoint you can use to submit txs.
It basically wraps ethereum rpc but overrides eth_sendRawTransaction to post to DA instead of exec client mempool